src/apis/price-updater.ts

```diff
@@ -0,0 +1,58 @@
+import axios from "axios";
+
+import { CurrencySymbol } from "../utils/currencies";
+
+// domain
+import { FiatExchangeRates, Tokens } from "../domain/hermez";
+
+interface ApiExchangeRate {
```

Comment:
This should go in the persistence level, at the end this is information coming from the outside that we need to convert to our domain (which you are already doing perfectly later).